min-turnup/phase2/ansible/Makefile

```diff
@@ -1,5 +1,6 @@
 TAG=v1
-IMAGE=gcr.io/mikedanese-k8s/install-k8s
+#IMAGE=gcr.io/mikedanese-k8s/install-k8s
+IMAGE=docker.io/colemickens/install-k8s
```

Comment:
Let's set this to `IMAGE ?= bar`, so it's easy to override with `make IMAGE=foo`.